src/components/tasks/CreateTask.jsx

```diff
@@ -0,0 +1,54 @@
+import { Fragment, useEffect, useRef, useState } from "react";
+import PropTypes from "prop-types";
+import sanitizeInput from "src/utils/helpers/sanitizeInput";
+import "src/components/tasks/CreateTask.component.scss";
+
+const CreateTask = ({ onAddTask }) => {
+  const [task, setTask] = useState("");
```

Comment:
I would use `null` instead of empty string

Reply:
I am getting this warning for using "null" instead of empty string.
`Warning: `value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.`

Reply:
Discussed.